packages/website/pages/_document.tsx

```diff
@@ -20,6 +35,9 @@ export default class MyDocument extends Document<{
             data-emotion-css={this.props.ids.join(' ')}
             dangerouslySetInnerHTML={{ __html: this.props.css }}
           />
+          {this.renderSnipet() && (
```

Comment:
call render `this.renderSnippet()` once and put it into a variable, use this variable to check the case and assign it to `__html`